.gitignore

```diff
@@ -198,7 +198,8 @@ obj/
 *.vsix
 *.zip
 
-
+# Api extractor
+packages/*/etc/
```

Comment:
Looks like the api-extractor config also uses a `temp` directory.

Reply:
we already exclude `temp` probably why I didn't notice

Reply:
Oh of course. Nevermind!